eventing/samples/cronjob-source/README.md

```diff
@@ -74,12 +74,19 @@ kubectl logs -l serving.knative.dev/service=message-dumper -c user-container --s
 
 You should see log lines showing the request headers and body from the source:
 
-```json
-{
-  "ID": "1543616460000180552-203",
-  "EventTime": "2018-11-30T22:21:00.000186721Z",
-  "Body": "{\"message\": \"Hello world!\"}"
-}
+```
+2019/03/14 14:28:06 Message Dumper received a message: POST / HTTP/1.1
+Host: message-dumper.default.svc.cluster.local
+Transfer-Encoding: chunked
+Accept-Encoding: gzip
+Ce-Cloudeventsversion: 0.1
```

Comment:
BTW: In the next release it will use 0.2.

Reply:
Thanks for the heads-up.

Reply:
I should dig out my code to switch this from something which dumps HTTP to something which dumps CloudEvents, since I think we want people talking about attributes/body, rather than HTTP framing.

But, this is correct at the moment, so let's get it in.